lang/java/reef-tests/src/test/java/org/apache/reef/tests/examples/TestHelloREEFMultiRuntime.java

```diff
@@ -48,16 +52,27 @@ public void tearDown() throws Exception {
   }
 
   @Test
-  public void testHelloREEFMultiRuntime() {
+  public void testHelloREEFMultiRuntime() throws InjectionException {
     if(this.testEnvironment instanceof YarnTestEnvironment){
       // multi runtime can be tested on yarn only
       final Configuration driverConf = DriverConfiguration.CONF
               .set(DriverConfiguration.GLOBAL_LIBRARIES, EnvironmentUtils.getClassLocation(this.getClass()))
-              .set(DriverConfiguration.DRIVER_IDENTIFIER, "TEST_HelloREEF")
+              .set(DriverConfiguration.DRIVER_IDENTIFIER, "TEST_HelloREEFMultiRunitme")
               .set(DriverConfiguration.ON_DRIVER_STARTED, HelloMultiRuntimeDriver.StartHandler.class)
               .set(DriverConfiguration.ON_EVALUATOR_ALLOCATED, HelloMultiRuntimeDriver.EvaluatorAllocatedHandler.class)
               .build();
-      final LauncherStatus state = this.testEnvironment.run(driverConf);
+
+      // create the multi runtime environment
+      Configuration multiruntimeConfig = new MultiRuntimeConfigurationBuilder()
+              .addRuntime(RuntimeIdentifier.RUNTIME_NAME)
+              .addRuntime(org.apache.reef.runtime.local.driver.RuntimeIdentifier.RUNTIME_NAME)
+              .setDefaultRuntime(RuntimeIdentifier.RUNTIME_NAME)
+              .setMaxEvaluatorsNumberForLocalRuntime(1)
+              .setSubmissionRuntime(RuntimeIdentifier.RUNTIME_NAME)
```

Comment:
On this level this is not main runtime, but a submission runtime. This is the cleint side config. MAin runtime is NOT a client side concept.